packages/astro/package.json

```diff
@@ -50,9 +50,9 @@
       "types": "./dist/index.d.ts",
       "import": "./dist/index.js"
     },
-    "./client/react": {
-      "types": "./dist/client/react/index.d.ts",
-      "import": "./dist/client/react/index.js"
+    "./react": {
+      "types": "./dist/react/index.d.ts",
+      "import": "./dist/react/index.js"
     },
     "./client/stores": {
```

Comment:
It makes sense since:

1. They will know it should only be used in `client`
2. We have a `store` suffix (i.e. `$authStore`) so they will know immediately that it is a nanostore